packages/web/src/scss/components/Stack/_Stack.scss

```diff
@@ -50,7 +50,6 @@
 }
 
 .Stack--hasIntermediateDividers > :first-child {
-    padding-block-start: 0;
```

Comment:
This line of the code shouldn't be part of the intermediate dividers styles – only in the combination with `Stack--hasSpacing` (included in the styles above). So I assume, this PR is bug fix, not BC, right?